1-exercises/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,20 @@
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+    arr.forEach(array);
```

Comment:
To make your codes neater, it's a very good practice to use conditional ternary operator instead of if else chaining. If you do so, you'll have one line of code instead of line 14 to 24